pkg/services/ngalert/provisioning/persist.go

```diff
@@ -42,6 +43,7 @@ type RuleStore interface {
 	UpdateAlertRules(ctx context.Context, rule []models.UpdateRule) error
 	DeleteAlertRulesByUID(ctx context.Context, orgID int64, ruleUID ...string) error
 	GetAlertRulesGroupByRuleUID(ctx context.Context, query *models.GetAlertRulesGroupByRuleUIDQuery) ([]*models.AlertRule, error)
+	CountInFolder(ctx context.Context, orgID int64, folderUID string, u *user.SignedInUser) (int64, error)
```

Comment:
It looks like this method is not used. I think we can remove it to not expand interfaces with unused methods.